base/files/file_unittest.cc

```diff
@@ -97,6 +98,7 @@ TEST(FileTest, Create) {
     EXPECT_EQ(base::File::FILE_ERROR_EXISTS, file.error_details());
```

Comment:
Do you know why did this part fail?

Reply:
Was it also because "win32 SbFileOpen doesn't fail on relative path"?

Reply:
I didn't look much into it and just took what was from trunk, but it looked to me like differences in our file opening error codes from base's